City-of-Helsinki/drupal-helfi-rekry
# UHF-5306: Job listing og tags

# UHF-5306: Job listing og tags

## What has been done
- removed the `og:image` tag implementation from code
- using the `og:shareable-image` custom tag defined in `hdbt_content` module
- added the missing tags in config

## How to install
1. set up REKRY instance
2. checkout this branch: `git checkout UHF-5306_job_listing_og_tags`
3. import config: `make drush-cim`
4. checkout equivalent branch of the theme: `composer require drupal/hdbt:dev-UHF-5306_job_listing_og_tags`
5. clear cache: `make drush-cr`

## How to test
- open any job listing node and check that
  - `og:image` tag gets its value in this priority order:
    1. node image, if one doesn't exist then use
    2. taxonomy term default image, if one doesn't exist then use
    3. site default og image
  - meta description tag gets its value from the "job description" field
  - rest of the og and other metatags are like they are in other content types

## Other PR's
- https://github.com/City-of-Helsinki/drupal-hdbt/pull/383
